### ISSUE TITLE
errors shown while scanning not shown after scan is complete

### ISSUE THREAD

**Issue by PromontoryProtean** [RickStrahl/WestWindWebSurge#64]
While scanning, WebSurge is saying that there were failed requests:
![websurge-scanning](https://user-images.githubusercontent.com/10098842/53964233-eb773c00-40bc-11e9-8f6b-bf8855160ee3.jpg)

After the scan completes, in both the results and output URL summary windows, WebSurge says no errors occurred :
![websurge-scanned](https://user-images.githubusercontent.com/10098842/53964296-0f3a8200-40bd-11e9-8973-3969f339c2a4.jpg)

If I look in my event log I do see the errors that were triggered while scanning, so I know that WebSurge was triggering errors in the site that I was testing. I know the docs say that results are limited to 1000 entries, but this scan stopped at 160 results as you can see, and even if I use the results filter to select errors instead of all, it just flips itself back to all and I see no errors.

I just discovered this app and it has already proven incredibly useful, but something doesn't seem right here. Am I missing something obvious?

**Comment by RickStrahl** [comment]
The results show no error which means the results that were actually logged don't have errors.

There's a warm up period (configurable) - is it possible the errors are occurring during that warm up phase so they don't show up int eh results?

**Comment by PromontoryProtean** [comment]
Ok, I found the WarmupSeconds setting and changed that to 0 and it started logging the errors. Ironically, I was using WebSurge to debug errors that were occurring during my app's startup sequence. I was wondering why my tests always ran for 2 seconds more than what I had entered in the time box, but didn't really think too much of it. I guess this WarmupSeconds setting explains that.

Thanks for the quick feedback. This app is incredibly useful. Planning to purchase a pro license today.
